shopsafe-backend/src/main/test-resources/WEB-INF/appengine-web.xml

```diff
@@ -7,7 +7,4 @@
     <!-- prevent unwanted caching when accessing via the web preview server -->
     <include path="/**" expiration="0s" />
   </static-files>
-  <resource-files>
-    <include path="/**" />
-  </resource-files>
```

Comment:
do we need this for the api key?

Reply:
I don't think so -- I think I just copied the files to have the right directory